plugin.xml

```diff
@@ -105,28 +100,16 @@
 
         <config-file target="AndroidManifest.xml" parent="/manifest/application">
             <activity
-                android:name=".wxapi.WXEntryActivity"
+                android:name=".wxapi.EntryActivity"
```

Comment:
Consolidated this into a single activity [here]() since we don't use `WXPayEntryActivity`.